frontend/src/app.config.ts

```diff
@@ -13,7 +13,7 @@ interface Config{
 }
 
 const config:Config = <Config>{};
-const hostname = process.env.CLOWDER_REMOTE_HOSTNAME || "";
+const hostname = process.env.CLOWDER_REMOTE_HOSTNAME || `${window.location.protocol}//${window.location.host}`;
```

Comment:
after another thought, maybe one file is enough. How about this?

Reply:
looks good. lets try it